runtime/doc/terminal.txt

```diff
@@ -185,6 +185,32 @@ clipboard.
 OSC 52 sequences sent from the :terminal buffer do not emit a |TermRequest|
 event. The event is handled directly by Nvim and is not forwarded to plugins.
 
+OSC 133: shell integration			*terminal-osc133*
+
+Some shells will emit semantic escape sequences (OSC 133) to mark the
+beginning and end of a prompt. The start of a prompt is marked with the
+sequence `OSC 133 ; A`. Nvim can be configured to create signs in terminal
+buffers marking shell prompts. Example: >lua
```

Comment:
Isn't the "end" sequence also important to mention? See https://github.com/neovim/neovim/pull/32771 for a docs update.